apis/bigqueryconnection/v1beta1/connection_reference.go

```diff
@@ -88,36 +87,13 @@ type BigQueryConnectionConnectionRef struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
-func parseExternal(external string) (string, string, string, error) {
+func ParseConnectionExternal(external string) (*parent.ProjectAndLocationParent, string, error) {
```

Comment:
Suggest this should return `(*ConnectionIdentity,error)`  (and not the parent split out)

Reply:
Sure. I don't have a preference. It used to be handy to split them because we don't have the identity.  😄